tests/Tests/100__deploy/110__auth_providers_and_rbac/110__rbac.robot

```diff
@@ -15,6 +15,17 @@ Verify Default Access Groups Settings And JupyterLab Notebook Access
     Verify Default Access Groups Settings
     Verify User Can Spawn A Notebook
 
+Verify Empty Group Doesnt Allow Users To Spawn Notebooks
+    [Documentation]   Verify that User is unable to Access Jupyterhub after modifying Access Groups in rhods-groups-config
```

Comment:
I think doc should specify that the user is unable to access if the groups are set as empty

Reply:
Updated.